src/js/engagement_view/src/services/uploadPlugin/deletePlugin.tsx

```diff
@@ -0,0 +1,16 @@
+import DEV_API_EDGES from '../constants';
+import {apiFetchPostRequest} from '../fetch';
+
+export const deletePlugin = async ( pluginName: string ): Promise <boolean> => {
+    const body = JSON.stringify( {plugins_to_delete: [pluginName]} );
+    
+    try{ 
+        const response = await apiFetchPostRequest(`${DEV_API_EDGES.modelPluginEdge}/deleteModelPlugin`, "post", body);
```

Comment:
it's sort of odd to me that every call of apiFetchPostRequest requires you specify "post" manually - might be better named `apiFetchWithBody` or something?

Reply:
Agreed & updated.